test/controllers/establishers/partnership/partner/details/CheckYourAnswersControllerSpec.scala

```
@@ -78,7 +78,7 @@ class CheckYourAnswersControllerSpec
   val commonJson: JsObject =
     Json.obj(
       "schemeName" ->schemeName,
-      "submitUrl"  -> "/migrate-pension-scheme/establisher/1/partnership/list-of-partners"
```

Approving this for jenkins failure, but we should always use reverse routes instead of hardcoding urls in such cases